website/docs/foundations/accessibility/index.md

```diff
@@ -1,64 +1,78 @@
 
-1. It adds a message to the page to let the screen reader user know that the route has changed and regular page navigation can resume (it is similar to [https://github.com/ember-a11y/a11y-announcer](https://github.com/ember-a11y/a11y-announcer) but does not use `aria-live`).
-2. It moves the focus to that message for the screen reader user, effectively resetting focus in Ember apps (similar to how a native web page/site works).
-3. It provides a bypass mechanism so the user can skip to the page’s primary content (see [https://www.w3.org/TR/WCAG20-TECHS/G1.html](https://www.w3.org/TR/WCAG20-TECHS/G1.html)). You can opt out of this if you want (see the `Options` section in the documentation for available options).
+1. It adds a message to the page to let the screen reader user know that the route has changed and regular page navigation can resume (it’s similar to [a11y-announcer](https://github.com/ember-a11y/a11y-announcer) but doesn’t use `aria-live`).
+2. It moves the focus to that message for the screen reader user, effectively resetting focus in Ember apps (similar to how a native web page works).
+3. It provides an optional bypass mechanism so the user can skip to the page’s primary content (see [Technique G1](https://www.w3.org/WAI/WCAG21/Techniques/general/G1)). See the `Options` section in the documentation for available options.
 
-## Page Organization
+## Page organization
 
-All content should be contained in elements with [landmark roles](https://www.w3.org/TR/wai-aria/#landmark_roles). Typically this will be the `header`, `main`, and `footer` elements, but depending on your page layout, you might also be using `aside` and `nav` as direct descendants of the `body` element as well.
+All content should be contained in elements with [landmark roles](https://www.w3.org/TR/wai-aria/#landmark_roles). Typically this includes the `header`, `main`, and `footer` elements, but depending on your page layout, you might also be using `aside` and `nav` as direct descendants of the `body` element.
 
-## Titles and Labels
+## Titles and labels
 
 ### Page titles
 
 Ensure that each page has a relevant page title. This is included in Ember applications by default, but older applications that have been updated over the years may need to manually add this in with [ember-page-title](https://github.com/ember-cli/ember-page-title).
 
 ### Headings
 
-Headings (h1-h6) should follow a logical order and not skip any levels. Dynamic applications can make this challenging, so this may require extra attention in checking what renders to the browser.
+Headings (`h1`-`h6`) should follow a logical order and shouldn’t skip any levels. Dynamic applications can make this challenging, so this may require extra attention in checking what renders to the browser.
```

Comment:
```suggestion
Headings (`h1`–`h6`) should follow a logical order and shouldn’t skip any levels. Dynamic applications can make this challenging, so this may require extra attention in checking what renders to the browser.
```

Reply:
Suggestion: Use en dash

https://www.thepunctuationguide.com/en-dash.html